userspace/falco/app_actions/open_inspector.cpp

```diff
@@ -67,14 +68,47 @@ application::run_result application::open_live_inspector(
 			// Regardless of the implementation, the underlying method remains the same.
 			inspector->open_udig();
 		}
-		else if(m_options.gvisor_config != "")
+		else if(!m_options.gvisor_config.empty())
 		{
 			falco_logger::log(LOG_INFO, "Enabled event collection from gVisor. Configuration path: " + m_options.gvisor_config);
 			inspector->open_gvisor(m_options.gvisor_config, m_options.gvisor_root);
 		}
-		else
+		else if(getenv(FALCO_BPF_ENV_VARIABLE) != NULL) /* BPF engine. */
+		{
+			try
+			{
+				const char *bpf_probe_path = std::getenv(FALCO_BPF_ENV_VARIABLE);
+				/* If the path is empty try to load the probe from the default path. */
+				if(strncmp(bpf_probe_path, "", 1) == 0)
+				{
+					const char *home = std::getenv("HOME");
+					if(!home)
+					{
+						return run_result::fatal("Cannot get the env variable 'HOME'");
+					}
+					char full_path[PATH_MAX];
+					snprintf(full_path, PATH_MAX, "%s/%s", home, FALCO_PROBE_BPF_FILEPATH);
+					bpf_probe_path = full_path;
+				}
+				inspector->open_bpf(2048, bpf_probe_path, m_state->ppm_sc_of_interest, m_state->tp_of_interest);
```

Comment:
TODO: `2048` will be a value parsed from Falco config; see #2164 ; it will be ported in this PR.

Moreover, IMHO i'd update `open_bpf` signature, putting the most important param, ie: `bpf_probe_path`, as first param; all the others should be optional ones.

A 0-valued `buffer_dimension` should enforce a default one in libscap.

Reply:
Done.